config/services.yml

```diff
@@ -575,6 +575,7 @@ https://cmr.uat.earthdata.nasa.gov:
       subsetting:
         bbox: true
         variable: true
+        shape: true
```

Comment:
Thanks Chris! We will want to add that eventually. Since this may require some testing can we move forward as-is so Jennifer can do her demo tomorrow?